extensions/amp-subscriptions/0.1/amp-subscriptions.css

```diff
@@ -107,6 +107,34 @@ html:not(.i-amphtml-subs-grant-yes) .i-amphtml-subs-dialog-close-button {
   display: none;
 }
 
+.subscriptions-progress {
```

Comment:
Let's make this a private class. E.g. `i-amphtml-subs-progress` or such. This way the authors can use their own completely self-authored indicator w/o colliding with our default styles.